src/routes/todoRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from "express";
+import { TodoController } from "../controllers/todoController";
+
+
+export function createTodoRouter(todoController: TodoController): Router{
+    const router = Router();
+    router.get("/", todoController.getTodos);
+    router.post("/", todoController.createNewTodo); //need to reed from req
+    //something that i have learned here is that in express routes are matched in the order they're defined :)
+    // so in the start i do the regular delete first and then every time i do delete all (/all) i got my message
+    // that this id isnt in the list.
+    router.delete("/all", todoController.clearTodosList)
+    router.delete("/:id", todoController.deleteTodo);
+
+    router.put("/:id", todoController.toggleTodo)
```

Comment:
according to rest protocol i thin it is better to use here patch here and not put